### DISCUSSION TITLE
下个版本能否多增加点图标啊。。。或者下个大版本增加也行啊

### DISCUSSION THREAD

**Discussion by hulang** [layui/layui#2286] [讨论]
有点少了。。。都是用阿里云的，应该不会侵权吧。。
目前还是使用：`font-awesome` 4.7+`layui`的，我是能用`layui`有的是不会使用：`font-awesome`。。。

**Comment by sentsim** [comment]
应该不会增加图标。包括后面的大版本，图标也将使用开源的。

**Reply by hulang** [reply]
明白了。。。
